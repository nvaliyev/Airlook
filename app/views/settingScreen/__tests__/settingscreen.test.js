import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SettingScreen from '..';

describe('SettingScreen', () => {
    test('SettingScreen snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<SettingScreen/>);
        expect(snapshot).toMatchSnapshot();
    });
});


