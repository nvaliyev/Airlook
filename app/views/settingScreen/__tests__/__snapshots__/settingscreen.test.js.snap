// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingScreen SettingScreen snapshot 1`] = `<SettingsContent />`;
